prov/verbs/src/fi_verbs.h

```diff
@@ -276,14 +276,14 @@ struct fi_ibv_eq_entry {
 typedef int (*fi_ibv_trywait_func)(struct fid *fid);
 
 struct fi_ibv_eq {
-	struct fid_eq		eq_fid;
-	struct fi_ibv_fabric	*fab;
-	fastlock_t		lock;
-	struct dlistfd_head	list_head;
-	struct rdma_event_channel *channel;
-	uint64_t		flags;
-	struct fi_eq_err_entry	err;
-	int			epfd;
+	struct fid_eq			eq_fid;
+	struct fi_ibv_fabric		*fab;
+	fastlock_t			lock;
+	struct dlistfd_head		list_head;
+	struct rdma_event_channel	*channel;
+	uint64_t			flags;
+	struct fi_eq_err_entry		err;
+	int				epfd;
```

Comment:
nit: Please remove this change.  White space changes cause issues with cherry picking patches for stable branches.